{{cookiecutter.project_slug}}/gulpfile.js

```diff
@@ -139,14 +139,14 @@ function initBrowserSync() {
       `${paths.js}/*.js`,
       `${paths.templates}/*.html`
     ], {
+      // https://www.browsersync.io/docs/options/#option-open
+      // Disable as it doesn't work from inside a container
+      open: false,
```

Comment:
Good catch, although we can let this option to true when not using Docker:

```suggestion
      {%- if cookiecutter.use_docker == 'y' %}
      // https://www.browsersync.io/docs/options/#option-open
      // Disable as it doesn't work from inside a container
      open: false,
      {%- endif %}
```